src/app/market/buy/checkout-process/checkout-process.component.html

```diff
@@ -252,11 +252,11 @@
             {{listing.basePrice?.getIntegerPart()}}{{listing.basePrice?.dot()}}<small>{{listing.basePrice?.getFractionalPart()}}</small>
             PART
           </td>
-          <td class="shipping cell">{{listing.internationalShippingPrice?.getIntegerPart()}}{{listing.internationalShippingPrice?.dot()}}<small>{{listing.internationalShippingPrice?.getFractionalPart()}}</small>
+          <td class="shipping cell">{{listing?.shippingAmount(valueOf('country')).getIntegerPart()}}{{listing?.shippingAmount(valueOf('country')).dot()}}<small>{{listing?.shippingAmount(valueOf('country')).getFractionalPart()}}</small>
```

Comment:
valueOf() has been removed.

Use this instead `shippingFormGroup.values.country || ''`